client/reader/style.scss

```diff
@@ -64,6 +64,15 @@ body.is-section-reader {
 			}
 		}
 
+		// For reader/search
```

Comment:
1. Added these styles here to keep all navigation-header styles at one place.
2. Removing bottom margin as now we are giving spacing to search fixed area.